apps/microtvm/zephyr/demo_runtime/src/main.c

```diff
@@ -265,6 +265,10 @@ static uint8_t main_rx_buf[RING_BUF_SIZE_BYTES];
 // The main function of this application.
 extern void __stdout_hook_install(int (*hook)(int));
 void main(void) {
+  // TODO (mehrdadh): Update this when zephyr version was updated to 2.6. 
+  // Uncomment this for qemu_riscv32, also update zephyr to latest version.
```

Comment:
Actually, can you just use the `#ifdef CONFIG_BOARD_QEMU_RISCV32`?

Reply:
that works, I think because it is an integer compare to using BOARD flag.